examples/linux-i2c.rs

```diff
@@ -0,0 +1,47 @@
+use ak09915_rs::Ak09915;
+use ak09915_rs::Mode;
+use clap::Parser;
+use linux_embedded_hal::I2cdev;
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+struct Args {
+    /// Receive the i2c device as parameter
+    #[arg(short, long, default_value_t = ("/dev/i2c-1").to_string())]
+    device: String,
+}
```

Comment:
Cool, are those commands automatically generated from the info contained in the cargo.toml file? Also, instead of author and about commands, wouldn't be better a `help` one?

Reply:
Helper is generated by default. Our projects were created before clap had support to structs directly, before that was another crate.